codeflash/api/cfapi.py

```diff
@@ -182,7 +183,7 @@ def get_blocklisted_functions() -> dict[str, set[str]] | dict[str, Any]:
     internal_server_error = 500
 
     owner, repo = get_repo_owner_and_name()
-    information = {"pr_number": pr_number, "repo_owner": owner, "repo_name": repo}
+    information = {"pr_number": pr_number, "repo_owner": owner, "repo_name": repo, "userId": get_user_id()}
```

Comment:
we dont want to pass the user_id here since the request has an api key embedded, which can be validated in the backend. getting user_id can also result in a cfapi request the first time.